sdk/search/search-documents/test/public/utils/setup.ts

```diff
@@ -233,29 +233,42 @@ export async function createIndex(
   };
 
   if (serviceVersion.includes("Preview")) {
-    const vectorSearchConfiguration = "algorithm-configuration";
+    const algorithm = "algorithm-configuration";
+    const vectorizer = "vectorizer";
+    const profile = "profile";
 
     hotelIndex.fields.push({
       type: "Collection(Edm.Single)",
       name: "vectorDescription",
       searchable: true,
       vectorSearchDimensions: 1536,
       hidden: true,
-      vectorSearchConfiguration,
+      vectorSearchProfile: profile,
     });
 
     hotelIndex.vectorSearch = {
-      algorithmConfigurations: [
+      algorithms: [
         {
-          name: vectorSearchConfiguration,
-          kind: "hnsw",
+          name: algorithm,
+          kind: "exhaustiveKnn",
```

Comment:
Any reason to change this from `hnsw` to `exhaustiveKnn`?

Reply:
It's a quick way to test that the new algorithm configuration works as intended. I'm thinking we consider more exhaustive testing as we move towards GA.